mail_attach_existing_attachment/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* The module only allows the addition of attachments linked to the object.
```

Comment:
nitpicking : shouldn't be in this file (it's more a description)

Reply:
in my sense it should stay there because the word "only" refers to a limitation that could be address in the future